### PR TITLE
Implementing changes in regression dialogue and subdialogue

### PR DESCRIPTION
**Pull request by stevekogo**
@dannyparsons @rdstern I have implemented what was proposed for simple regression dialogue but here are a few concerns.

Should the response variable receiver be restricted to non-factor columns? In case of a factor, it returns an error if you intend to display estimates and ANOVA table.

We are still missing implementation of prediction interval for the fitted model in graphics tab. Can Beth send me the code for this?

Lastly, we can run multiple scripts but for graphics, it cannot diaplay multiple plots. What should we do about this?

### CODE REVIEW

**Comment by dannyparsons** [comment]
Hi Steve, these are interesting points you've raised. This is the level of detail we need to get things complete.

1. Yes, we should restricted to non-factor columns. Currently we can't do that going through Instat object. We only have a method to get columns of a certain type. So we should think how best to modify that R method to get columns not of a certain type. And generally we may as well think how to get columns of type A or type B, or columns not of type A or type B etc. I'll write it as an issue, we could work on it together if you like.

2. Yes, Beth and Roger have discussed this but apparently it is not that easy to do. When they have a solution they'll send it over to you, but because it is quite complex, it's not a priority at the moment.

3. Another good issue. We need to think generally how to display multiple plots. This could link with what @deaspo has been looking at with the Output Window. We may want to display graphs in the output window. We should also look at the RLink to see if there is a way to get multiple graphs through RDotNet. This needs to be fixed generally, so happy to keep your dialog as it is and until we fix the graphics, you can only see one plot.

File: instat/sdgSimpleRegOptions.vb [hunk 1]
```diff
@@ -99,78 +98,79 @@ Public Class sdgSimpleRegOptions
...
+    Private Sub pvalues()
+        If (chkANOVA.Checked Or chkEstimates.Checked) Then
+            chkPvalues.Enabled = True
         Else
-            chkAnovaPvalues.Checked = False
-            chkAnovaPvalues.Enabled = False
+            chkPvalues.Checked = False
+            chkPvalues.Enabled = False
         End If
+    End Sub
```
Comment (dannyparsons) [inline instat/sdgSimpleRegOptions.vb]: I can see what you are thinking here but I don't think we need to be this clever. We should have pvalues checked by default, but if the user turns it off, that's fine and it stays off unless they tick it again. So we can remove this method I think.

**Comment by rdstern** [comment]
Hi Steve,

Just to say that Danny’s first comment can be broken into 2 separate points.


a)      In the regression the first receiver should be a number column.  So that restriction can be made by you.

b)      In general, with the 3 main types (number, factor, string/character), we do want to add a possible restriction of not a particular type.  Most receivers will be “not string”.

Roger

File: instat/sdgSimpleRegOptions.vb (hunk 1 above)
Comment (stevekogo) [inline instat/sdgSimpleRegOptions.vb]: But both ANOVA and Estimates are checked by default and the user will turn p_values off by unchecking it or unchecking both ANOVA and Estimates.

Comment (dannyparsons) [inline instat/sdgSimpleRegOptions.vb]: I'm happy with this as long as you don't do `Checked = False` when it's disabled. 
They way you have it now, when it becomes enabled again, the user has to check Pvalues again (or may forget). Which I don't like as much. But happy for it to be disabled in it's current state if neither ANOVA or Estimates is checked.

**Comment by stevekogo** [comment]
@dannyparsons Look at how I have fixed the p-values check and uncheck

**Pull request merged**